### DISCUSSION TITLE
Awesome한 시작 페이지에 관한 토론

### DISCUSSION THREAD

**Discussion by JunMango** [JustHeading/issue#5] [Ideas]
댓글로 브레인 스토밍이란 걸 해볼까?

**Comment by BurningFalls** [comment]
server 쪽은 해야 하는게 명확해서 자료 만드는건 상관없는데, 웹 디자인 구상이 문제긴 하네

**Reply by JunMango** [reply]
아 그냥 이랬으면 좋겠다 요런 느낌 원하는거 있나해서~ 문제는 아니야
